### DISCUSSION TITLE
DISABLE_AUTH & DISABLE_REGISTRATION do not disable auth nor prevent registration

### DISCUSSION THREAD

**Discussion by lucacastelnuovo** [plausible/analytics#2529] [Self-Hosted Support] [answered]
This is my setup: https://github.com/Luca-Castelnuovo/docker.ltc/tree/master/stats.

If I delete the `data` folder and recreate the containers, I get prompted to register.
After my registration, auth is still required.

How can I disable this?

**Comment by ruslandoga** [comment, marked as answer]
@Luca-Castelnuovo 👋

`DISABLE_AUTH` is no longer supported starting in [`v1.5`](https://github.com/plausible/analytics/discussions/2507)

**Reply by lucacastelnuovo** [reply]
Thank you for letting me know.
The option is still listed here: https://plausible.io/docs/self-hosting-configuration#server
